internal/gazelle.bash.in

```diff
@@ -62,12 +62,18 @@ function set_goroot {
 
 # If arguments were provided on the command line, either replace or augment
 # the generated args.
-if [ "${1-}" = "-args" ]; then
-  shift
-  ARGS+=("$@")
-elif [ $# -ne 0 ]; then
-  ARGS=("$@")
-fi
+case "${1-}" in
+  "fix" | "update" | "help" | "update-repos")
+    ARGS=("$@")
+    ;;
+  "-args")
```

Comment:
@jayconrod do we need to support this case? It's undocumented and only used by autogazelle, which should be easy to update

Reply:
Should be fine to drop or change. I don't think it works very well now.